From 5c3ee204aa61c35467e857b1aa1be4ff1c41729d Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Wed, 06 Nov 2024 11:17:49 +0200
Subject: [PATCH] [SW-207762] fs1: remove write to vuart

VUART in fs1 is different than gaudi, and will
use FIFO mechanism, hence current implementation
is removed.

Change-Id: I872d74ff5a66f6afe480aa824711e8c62ebff74a
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 74ed06c..7a7ad17 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -228,8 +228,6 @@
 
 __weak void preboot_putc_virt_uart(const char c)
 {
-	regmap_writel(c, mmVUART_MNG0_TX_DATA);
-	regmap_writel(1, mmVUART_MNG0_TX_STATUS);
 }
 
 #define EARLY_WRITE_RESP_MASK	(ARC_LOCAL_CONF_ARC_LBU_AXI_SPLIT_CTRL2_FORCE_WR_BUF_M |	\
